files-community/Files
# Fix: Fixed issue that caused shortcut keys to be displayed incorrectly in the command palette

**Resolved / Related Issues**
   Closes #issue...

**Validation**
How did you test these changes?
   1. Type '>' into the path bar.
   2. Type a backspace. Shortcut keys will disappear.
   3. Type '>' again. Shortcut keys will appear correctly.
